example/SimpleApp/src/main.cpp

```diff
@@ -305,8 +306,18 @@ static void loadSettings()
      */
     bool status = preferences.begin(PREF_NAMESPACE, true);
 
-    /* Settings found? */
-    if (true == status)
+    /* Settings not found? */
+    if (false == status)

```

Comment:
We can avoid this here by assigning the default values to the variables in the upper section. This saves again some space.